Core/Core.csproj

```diff
@@ -1,21 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <Version>1.0.0</Version>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Lykke.Service.Affiliate.Client" Version="1.0.7" />
     <PackageReference Include="Lykke.Service.AssetDisclaimers.Client" Version="3.0.0" />
     <PackageReference Include="Lykke.Service.CandlesHistory.Client" Version="2.1.0" />
     <PackageReference Include="Lykke.Service.ClientDialogs.Client" Version="1.0.0" />
+    <PackageReference Include="Lykke.Service.ClientAccountRecovery.Client" Version="1.0.17-beta" />

```

Comment:
Package would be updated after stable nuget release of Lykke.Service.ClientAccountRecovery.

Reply:
Updated